portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -3550,6 +3551,10 @@ private void _performDynamicQueryActions(
 		ActionableDynamicQuery actionableDynamicQuery =
 			getActionableDynamicQuery();
 
+		actionableDynamicQuery.setInterval(10);
```

Comment:
@manuelRives, we have to sort this 😢.

setAddCriterua > setInterval > setPerformActionMethid > setTransactionConfig

Reply:
Completely true, my bad. 
Fixed.